protocol/requests/create_account.go

```diff
@@ -12,27 +12,43 @@ var ErrCreateAccountInvalidBackupDisabledDataDir = errors.New("create-account: i
 var ErrCreateAccountInvalidLogFilePath = errors.New("create-account: invalid log file path")
 
 type CreateAccount struct {
-	DisplayName        string `json:"displayName"`
-	Password           string `json:"password"`
-	ImagePath          string `json:"imagePath"`
-	CustomizationColor string `json:"customizationColor"`
 	// RootKeystoreDir is the directory where keys are stored
 	RootKeystoreDir string `json:"rootKeystoreDir"`
 	// BackupDisabledDataDir is the directory where backup is disabled
 	BackupDisabledDataDir string `json:"backupDisabledDataDir"`
```

Comment:
not relate to your PR, but can we rename it to `RootDataDir` ? It seems they are equal and I prefer `RootDataDir`(the base dir(absolute path) application used) since it looks more clear